.github/workflows/rebase.yaml

```diff
@@ -0,0 +1,21 @@
+name: rebase
+
+on:
+  pull_request:
+    types: [opened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  rebase:
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase') && (github.event.comment.author_association == 'CONTRIBUTOR' || github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER')
```

Comment:
idk if there's a `startsWith`?

> If you want you can rebase this w `/rebase`

-> 💥, anyway. Lets see how it goes!

Reply:
This action is used in Flux for some months now and it works as long as the PR doesn't make changes to the workflows dir.